packages/core/src/components/Dropdown/Dropdown.types.ts

```diff
@@ -87,6 +87,18 @@ export interface DropdownComponentProps extends CustomMenuBaseProps, CustomOptio
    * Called when blurred
    */
   onBlur?: FocusEventHandler;
+  /**
+   * Called when scrolling
+   */
+  onScroll?: (event: any) => void;
```

Comment:
it is suppose to be a regular HTML handler, so probably:
```suggestion
  onScroll?: UIEventHandler<HTMLDivElement>;
```